linux/mcompile/keymap/mc_import_rules.cpp

```diff
@@ -439,61 +413,59 @@ bool KMX_ImportRules(LPKMX_KEYBOARD kp, vec_dword_3D& all_vector, GdkKeymap **ke
   //-------------------------------------------------------------
 
   int nDeadkey = 0;
-  LPKMX_GROUP gp = new KMX_GROUP[kp->cxGroupArray+4];  // leave space for old
+  LPKMX_GROUP gp = new KMX_GROUP[kp->cxGroupArray + 4];  // leave space for old
   memcpy(gp, kp->dpGroupArray, sizeof(KMX_GROUP) * kp->cxGroupArray);
 
   //
   // Find the current highest deadkey index
   //
 
   kp->dpGroupArray = gp;
-  for(UINT i = 0; i < kp->cxGroupArray; i++, gp++) {
-    //if(gp->fUsingKeys && gp->dpNoMatch == NULL) {   // I4550
-     // WCHAR *p = gp->dpNoMatch = new WCHAR[4];
-     // *p++ = UC_SENTINEL;
-     // *p++ = CODE_USE;
-     // *p++ = (WCHAR)(kp->cxGroupArray + 1);
-     // *p = 0;
+  for (UINT i = 0; i < kp->cxGroupArray; i++, gp++) {
+    // if(gp->fUsingKeys && gp->dpNoMatch == NULL) {   // I4550
+    //  WCHAR *p = gp->dpNoMatch = new WCHAR[4];
+    //  *p++ = UC_SENTINEL;
+    //  *p++ = CODE_USE;
+    //  *p++ = (WCHAR)(kp->cxGroupArray + 1);
+    //  *p = 0;
```

Comment:
Do you still need the commented code?

Reply:
OK-deleted even  though it is commented out in the windows version